### PR TITLE
1host.js CLI

### LINKED ISSUE #57: [Feature Request] cli options
**Is your feature request related to a problem? Please describe.**
no
**Describe the solution you'd like**
add cli options like
- start
- start in a different directory
- config
- help
- etc.
**Describe alternatives you've considered**
manually doing this
**Additional context**
examples:
```bash
1host help
1host start
1host start --location=./tests/posix/
1host config
```

### PR DESCRIPTION
**Pull request by wgyt**
**Resolves**

Resolves #57

**Changes**

Added more cli comments

**Reason for changes**

Better cli that makes life easier

**Tests**

none, yet.

### CODE REVIEW

**Comment by wgyt** [comment]
This is almost done 🎉
I just need to make adding modules possible, right now it is a todo

File: bin/help.mjs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  formatting.successBox(
+    l10n(
+      `Commands:
+        > start: starts 1host.js
+          >> args: directory: changes the directory
+          >> usage: 1host start < directory (optional) >
+        > config: changes the config file
+          >> args: directory: changes the directory
+          >> usage: 1host config < directory (optional) >
+        `,
```
Comment (GrahamSH-LLK) [inline bin/help.mjs:16]: Put this in strings

File: bin/config.mjs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      } else {
+        readline.close();
+      }
+    });
+  }
+  e();
+  const dir = process.argv[3] || process.cwd();
+  fs.writeFileSync(
+    path.join(dir, "1host.config.js"),
+    `module.exports={port:${portt},modules:${modules}}`,
```
Comment (GrahamSH-LLK) [inline bin/config.mjs:49]: It should read the json and change it using JS

**Review by GrahamSH-LLK — CHANGES_REQUESTED** [review]

File: bin/config.mjs (hunk 2 above)
Comment (wgyt) [reply bin/config.mjs:49]: Right now it only makes the config file :\ but I guess I could change it

File: bin/help.mjs (hunk 1 above)
Comment (wgyt) [reply bin/help.mjs:16]: how do I do that?

**Comment by wgyt** [comment]
the config utility is almost done but it won't work with modules yet, but we give instructions. To be fair we warn them so
🤷🏻‍♂️

**Review by retronbv — CHANGES_REQUESTED** [review]
> the config utility is almost done but it won't work with modules(fully) yet

Fix that then it’s good

**Comment by wgyt** [comment]
> > the config utility is almost done but it won't work with modules(fully) yet
> 
> Fix that then it’s good

Can you fix it? I wasnt able to get it to work :(

**Comment by wgyt** [comment]
@GrahamSH-LLK I figured out how it broke, it executes the code after the function while the function is running 🐢

**Comment by retronbv** [comment]
> > > the config utility is almost done but it won't work with modules(fully) yet
> > 
> > 
> > Fix that then it’s good
> 
> Can you fix it? I wasnt able to get it to work :(

I don’t do that kind of stuff

**Comment by wgyt** [comment]
@retronbv @GrahamSH-LLK IT WORKS!!!! LETS GOOOOOOOOOO

**Pull request merged**